### PR TITLE
feat: Add `symptomatic` as return option for SGSS

### LINKED ISSUE #581: Add `Symptomatic` column as a return option for SGSS
The `AllTests` versions of the SGSS tables contain a `Symptomatic` column, which [3 different users](https://ebmdatalab.slack.com/archives/C010SJ89SA3/p1623410097410800) have identified as being important for their studies.

It's not clear from the [schema](https://github.com/opensafely/database-notebooks/blob/master/notebooks/database-schema.ipynb) what exactly this contains, but it's quite likely that the raw column would be acceptable to return.

### PR DESCRIPTION
**Pull request by StevenMaude**
Fixes #581; see that issue for more context.

### CODE REVIEW

**Review by sebbacon — COMMENTED** [review]
I realise this is draft so you're probably already planning it, but just a reminder that docstrings will also need updating per Alex's observation _I think either way of doing it would be okay, so long as it's clearly documented_

**Comment by sebbacon** [comment]
Another thing in passing - probably adding to `news.md` in the docs?

File: tests/test_tpp_backend.py [hunk 1]
```diff
@@ -2324,6 +2349,9 @@ def test_patients_with_test_result_in_sgss():
         sgtf_result=["9", "1", "", ""],
         variant=["B.1.351", "VOC-20DEC-01", "", ""],
         variant_detection_method=["Private Lab Sequencing", "Reflex Assay", "", ""],
+        symptomatic_positive=["Y", "N", "", ""],
+        symptomatic_negative=["N", "", "", ""],
+        symptomatic_any=["N", "", "", ""],
```
Comment (madwort) [inline tests/test_tpp_backend.py:2354]: am I misunderstanding this, or should this be 
```
symptomatic_any=["N", "N", "", ""],
```

Comment (madwort) [reply tests/test_tpp_backend.py:2354]: oh, no, I see - the `SGSS_AllTests_Negative` for this patient is first

**Review by inglesp — APPROVED** [review]
I suspect `+110 -14` doesn't do justice to the amount of work that's gone into this.  Chapeau.

File: cohortextractor/tpp_backend.py [hunk 2]
```diff
@@ -1830,6 +1830,25 @@ def patients_with_test_result_in_sgss(
...
             column = "t2.variant_detection_method"
+        elif returning == "symptomatic":
+            raw_column = "t2.symptomatic"
+            transforms = {
+                # From SGSS_AllTests_Positive table.
+                "N": "N",
+                "U": "",
+                "Y": "Y",
+                # From SGSS_AllTests_Negative table.
+                "NULL": "",
```
Comment (inglesp) [inline cohortextractor/tpp_backend.py:1841]: Are you matching the string `"NULL"` or a literal SQL `NULL`?

**Review by inglesp — COMMENTED** [review]
Actually, sorry, I've just spotted a potential issue with NULLs -- thoughts?

File: cohortextractor/tpp_backend.py (hunk 2 above)
Comment (StevenMaude) [reply cohortextractor/tpp_backend.py]: Think I've fixed this now.

**Comment by StevenMaude** [comment]
@inglesp: please take another look :eyes:

**Review by inglesp — APPROVED** [review]
That's better, thanks!

**Pull request merged**